Felienne/hedy
# [BUG FIX] Keeps shared status on program overwrite

**Description**
In this PR we implement a fix so we keep the current shared status of a program when overwriting. When a user continues working on a program and tries to save a `duplicate-check` modal is returned. Currently when deciding to overwrite we only keep the `program_id` but nothing else. In this PR we also copy the shared status to the new program.

**Fixes #1893**

**How to test**
Verify that this works correctly in two ways. Make sure you have a saved program, open it and over-write the program with the same name (simply press save). Notice that the shared status remains. The other way around; verify that a non-shared program stays non-shared when overwriting it.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
